README.md

```diff
@@ -1,9 +1,9 @@
-<a href="http://valor-software.com/ngx-bootstrap/#/">
-    <h1 align="center">ngx-bootstrap</h1>
-</a>
+<p align="center">
+  (<a href="http://valor-software.com/ngx-bootstrap/">ngx-bootstrap</a>)
+</p>
 
 <p align="center">
-Best way to quickly integrate <a href="https://getbootstrap.com/">Bootstrap 3</a> or <a href="https://getbootstrap.com/docs/4.0">Bootstrap 4</a> Components with <a href="https://angular.io/">Angular</a>
+Best way to quickly integrate <a href="https://getbootstrap.com/">Bootstrap 3</a> or <a href="https://getbootstrap.com/docs/4.0">Bootstrap 4</a> Components with <a href="https://angular.io/"</a>
```

Comment:
Any reason for this change? 
btw, `Components with <a href="https://angular.io/"</a>` this code doesn't work :)